viewing_party/party.py

```diff
@@ -1,23 +1,151 @@
 # ------------- WAVE 1 --------------------
 
+# I feel like there's a better way to test if title, genre, & rating are truthy 
+# but not sure how
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {"title": title, "genre": genre, "rating": rating}
+    if title and genre and rating:
+        return new_movie
+    else:
+        return None
+
+
+def add_to_watched(user_data, movie):
+    has_watched = [movie] 
+    user_data = {"watched" : has_watched }
+    return user_data
+
 
+def add_to_watchlist(user_data, movie):
+    wants_to_watch = [] 
+    user_data = {"watchlist" : wants_to_watch }
+    wants_to_watch.append(movie)
+
+    return user_data
+
+def watch_movie(user_data, title):
+    for movies in user_data["watchlist"]:
+        if movies["title"] == title:
+            user_data["watchlist"].remove(movies)
+            user_data["watched"].append(movies)
+    return user_data
+        
+        
 # -----------------------------------------
-# ------------- WAVE 2 --------------------
 # -----------------------------------------
+# ------------- WAVE 2 --------------------
+# -----------------------------------------        
+
+def get_watched_avg_rating(user_data):
+    count = 0
+    movie_rating = 0.0
+    if len(user_data["watched"]) == 0:

```

Comment:
Recall that an empty list is Falsy. Since that's the case, it's more Pythonic to write:
```python
if not user_data["watched"]:
    return 0.0
```

and since there's a possibility that watched list is empty and we'll just return 0.0, it makes more sense to have the variables `count` and `movie_rating` come after your guard clause on line 44. 

There's no need to create variables that don't get used when we just return 0.0

Additionally, since you have a guard clause then you don't need to put your logic in an else block. Since you'll have already passed the check to confirm that watched list has valid values on line 43, you can delete `else` and un-indent lines 46-49